cookbook/test/integration/localzk/serverspec/agent_spec.rb

```diff
@@ -0,0 +1,13 @@
+# Encoding: utf-8
+require 'spec_helper'
+
+describe 'Services' do
+  [8088, 8882].each do |p|
+    describe port(p) do
+      it { should be_listening.with('tcp') }
+    end
+  end
+  describe port(2181) do
+    it { should be_listening.with('tcp6') }
+  end
```

Comment:
Might be good to add tests to assert that the agent service is running:

```ruby
describe service 'baragon-agent' do
  it { is_expected.to be_enabled }
end

describe service 'baragon-agent' do
  it { is_expected.to be_running }
end
```

Also, the `is_expected.to` syntax is the preferred now with Rspec 3 (which Serverspec 2 uses).

You might also write those assertions above like so:

```ruby
it 'has a running baragon-agent service' do
  expect(service('baragon-agent')).to be_running
end
```